test/simplify_qdq_test.cpp

```diff
@@ -0,0 +1,484 @@
+        auto c1   = m1.add_instruction(migraphx::make_op("convolution",
+                                                       {{"padding", {0, 0, 0, 0}},
+                                                        {"stride", {1, 1}},
+                                                        {"dilation", {1, 1}},
+                                                        {"group", 1},
+                                                        {"padding_mode", 0}}),
+                                     d5,
+                                     d1);
+        auto mb13 = m1.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 1280, 7, 7}}}), l14);
+        auto mb14 = m1.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 1280, 7, 7}}}), l4);
+        auto q2   = m1.add_instruction(migraphx::make_op("quantizelinear"), c1, mb13, mb14);
+        auto mb15 = m1.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 1280, 7, 7}}}), l14);
+        auto mb16 = m1.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 1280, 7, 7}}}), l4);
+        auto d6 = m1.add_instruction(migraphx::make_op("dequantizelinear"), q2, mb15, mb16);
+        m1.add_return({d6});
+    }
+
+    migraphx::module m2;
+    {
+        auto pa  = m2.add_parameter("pa", s7);
+        auto p3  = m2.add_parameter("p3", s4);
+        auto l4  = m2.add_literal(s1, ints[1]);
+        auto l5  = m2.add_literal(s1, ints[2]);
+        auto l14 = m2.add_literal(s8, floats[5]);
+        auto l15 = m2.add_literal(s8, floats[6]);
+
+        auto mb9 = m2.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 320, 7, 7}}}), l15);
+        auto mb10 = m2.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 320, 7, 7}}}), l5);
+        auto q1   = m2.add_instruction(migraphx::make_op("quantizelinear"), pa, mb9, mb10);
+        auto c1   = m2.add_instruction(migraphx::make_op("quant_convolution",
+                                                       {{"padding", {0, 0, 0, 0}},
+                                                        {"stride", {1, 1}},
+                                                        {"dilation", {1, 1}},
+                                                        {"group", 1},
+                                                        {"padding_mode", 0}}),
+                                     q1,
+                                     p3);
+        auto mb15 = m2.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 1280, 7, 7}}}), l14);
+        auto mb16 = m2.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 1280, 7, 7}}}), l4);
+        auto d6 = m2.add_instruction(migraphx::make_op("dequantizelinear"), c1, mb15, mb16);
+        m2.add_return({d6});
+    }
+
+    migraphx::simplify_qdq sqdq;
+    sqdq.apply(m1);
```

Comment:
Most of the time we add a `run_pass` function in the test file to do this:

```cpp
void run_pass(migraphx::module& m)
{
    migraphx::simplify_qdq sqdq;
    sqdq.apply(m);
}
```

So we have a central place to update how the pass is applied.